helix-term/src/ui/picker.rs

```diff
@@ -266,8 +266,8 @@ pub struct Picker<T> {
     options: Vec<T>,
     // filter: String,
     matcher: Box<Matcher>,
-    /// (index, score)
-    matches: Vec<(usize, i64)>,
+    /// (index, score, highlight indices)
+    matches: Vec<(usize, i64, Option<Vec<usize>>)>,
```

Comment:
Let's get rid of this cache, it significantly increases memory consumption if the picker has a lot of entries. Fuzzy matching is reasonably fast enough to do at `render()` time with no cache.

Reply:
Done.